src/dev_query_arweave.erl

```diff
@@ -297,9 +297,11 @@ commitment_id_to_base_id(ID, Opts) ->
 all_ids(ID, Opts) ->
     Store = hb_opts:get(store, no_store, Opts),
     case hb_store:list(Store, << ID/binary, "/commitments">>) of
-        {ok, []} -> [ID];
-        {ok, CommitmentIDs} -> CommitmentIDs;
-        _ -> []
```

Comment:
I didn't see a use case to return an empty array. The LMDB store didn't return `not_found`, but the `hb_store_fs` did, so swapping these two stores in the `dev_query_test_vectors:simple_blocks_query_test` will make the test fail (which shouldn't happen).